packages/merge-on-green/test/merge-on-green.ts

```diff
@@ -470,7 +470,7 @@ describe('merge-on-green-', () => {
               repo: 'testRepo',
               number: 1,
               owner: 'testOwner',
-              created: Date.now() - 10800000,
+              created: Date.now() - 10920000,
```

Comment:
I kinda have to guess what that number means - would be great to add a comment explaining it's (looks at calculator) a little over 3 hours?

Reply:
Ah, will do! Basically, it limits the time interval in which it comments to 4 minutes, which is the period between runs for cloud scheduler. This *approximates* that it will comment only once, but I'll add some comments!